frontend/src/Interface/DeveloperInterface.tsx

```diff
@@ -0,0 +1,5 @@
+export interface DeveloperInterface {
+    id: number;
+    first_name: string;
+    last_name: string;
+  }
```

Comment:
Indentation still a little wonky here. Locally my ESLint corrected it, so you probably did this step before correcting the ESLint stuff :)